src/renderers/dom/shared/findDOMNode.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule findDOMNode
+ * @flow
+ */
+
+let findFiber = function(arg) {
+  return null;
+};
+let findStack = function(arg) {
+  return null;
+};
+
+const findDOMNode = function(componentOrElement : Element | ?ReactComponent<any, any, any>) : null | Element | Text {
+  return findStack(componentOrElement) || findFiber(componentOrElement);
```

Comment:
I think `findStack` would throw if this is a fiber because it would find it in the instance map and then trigger this invariant: https://github.com/facebook/react/blob/master/src/renderers/dom/stack/client/findDOMNode.js#L53-L60